README.md

```diff
@@ -79,6 +77,33 @@ The current implementation uses goroutines very liberally which means the query
 
 The current implementation creates a physical plan directly from the PromQL abstract syntax tree. Plan optimizations not yet implemented and would require having a logical plan as an intermediary step.
 
+## Distributed execution mode
+
+The supports a distributed mode where aggregations can be delegated to multiple remote engines, each responsible for an independent dataset. This mode is currently implemented through an optimizer which rewrites a query as a combination of multiple remote and one local aggregation. For example, when two remote engines are available, a query like:
```

Comment:
Randomly noticed, but it probably should read `The engine supports`.

Reply:
Nice catch, thanks.